AnswerKingAPI.yml

```diff
@@ -213,6 +264,33 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProblemDetails'
+    post:
+      tags:
+        - Inventory
+      summary: Unretire a retired category.
+      parameters:
+        - name: id
+          in: path
+          description: ''
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '201':
+          description: Created.
```

Comment:
Same as for the tags - should be more descriptive and return the newly-unretired category

Reply:
changed